models/issues/label.go

```diff
@@ -109,6 +111,15 @@ func (l *Label) CalOpenIssues() {
 	l.NumOpenIssues = l.NumIssues - l.NumClosedIssues
 }
 
+// Archived set the label as archived
+func (l *Label) Archived(isArchived bool) {
+	if isArchived {
```

Comment:
```suggestion
	if isArchived && l.ArchivedUnix.IsZero() {
```
Otherwise, a label will suddenly get a new archival time when you edit another attribute that is not the archival time.